app/views/facilities/manage.html.haml

```diff
@@ -10,6 +10,9 @@
   = f.input :short_description
   = f.input :description
   = f.input :banner_notice if SettingsHelper.feature_on?(:facility_banner_notice)
+  - if SettingsHelper.feature_on?(:facility_tile_list)
```

Comment:
Can we change this to `- if SettingsHelper.feature_on?(:facility_tile_list) && @facility.thumbnail?` so it only appears when there is a thumbnail? I'm seeing the same broken image here.